libs/common/src/models/export/cipher.export.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { CipherExport } from "@bitwarden/common/models/export/cipher.export";
+import { SecureNoteExport } from "@bitwarden/common/models/export/secure-note.export";
+import { CipherType } from "@bitwarden/common/vault/enums";
+import { CipherView } from "@bitwarden/common/vault/models/view/cipher.view";
+
+describe("Cipher Export", () => {
+  describe("toView", () => {
+    it("should preserve existing date values when request dates are null", () => {
+      const existingView = new CipherView();
+      existingView.creationDate = new Date("2023-01-01T00:00:00Z");
+      existingView.revisionDate = new Date("2023-01-02T00:00:00Z");
+      existingView.deletedDate = new Date("2023-01-03T00:00:00Z");
+
+      const request = CipherExport.template();
+      request.type = CipherType.SecureNote;
+      request.secureNote = SecureNoteExport.template();
```

Comment:
🎨 We should prefer `undefined` to `null`, which means we need to test both `null` and `undefined` in the test. That'll ensure that when we go to remove `null` from the interface, that we can rely on its counterpart.

```suggestion
    it.each([[null], [undefined]])("should preserve existing date values when request dates are nullish (=%p)", (date) => {
      const existingView = new CipherView();
      existingView.creationDate = new Date("2023-01-01T00:00:00Z");
      existingView.revisionDate = new Date("2023-01-02T00:00:00Z");
      existingView.deletedDate = new Date("2023-01-03T00:00:00Z");

      const request = CipherExport.template();
      request.type = CipherType.SecureNote;
      request.secureNote = SecureNoteExport.template();
      request.creationDate = date;
      request.revisionDate = date;
      request.deletedDate = date;
```

⛏️ You also need a test case checking that the values from `req` are copied when they're non-nullish.

Reply:
Good call! Updated in [0cf4363](https://github.com/bitwarden/clients/pull/15993/commits/0cf4363d88e810fdbd25dac3614cc9ffcb9a0a1c) and [091c251](https://github.com/bitwarden/clients/pull/15993/commits/091c251a40225c97a1b49050c136a26697faaa6b)